devito/types/basic.py

```diff
@@ -867,7 +867,7 @@ def symbolic_shape(self):
         halo = [sympy.Add(*i, evaluate=False) for i in self._size_halo]
         padding = [sympy.Add(*i, evaluate=False) for i in self._size_padding]
         domain = [i.symbolic_size for i in self.dimensions]
-        ret = tuple(sympy.Add(i, j, k, evaluate=False)
+        ret = tuple(sympy.Add(i, j, k)
```

Comment:
out of curiosity, why? it helped visually separating the left halo/padding from the right halo/padding

Reply:
Because you ended up with explicit `[i_size + 0 + 0 + 0 + 0]` all the way to the generated c code.

Reply:
is that the same in master too?

Reply:
It's not no, it breaks some tests with the `evaluate=False`